moonraker/components/notifier.py

```diff
@@ -139,9 +141,25 @@ async def notify(self, event_name: str, event_args: List) -> None:
             event_name if self.body is None else self.body.render(context)
         )
 
+        # Verify the attachment
+        if self.attach is not None:
+            if not self.attach.startswith("http://") and \
+               not self.attach.startswith("https://"):
+                fm = self.server.lookup_component("file_manager")
+                if not fm.can_access_path(self.attach):
+                    if not self.warned:
+                        self.server.add_warning(
+                            f"""Attachment of notifier '{self.name}' is not
+                            valid. The location of the
+                            attachment is not
+                            accessible.""")
```

Comment:
We don't want to use a multi-line string here, as newlines and leading whitespace is included.  So this would evauate to something like:
```
Attachment of notifier '{self.name}' is not\n                        valid. The location of the\n                        attachment is not\n                        accessible.
```